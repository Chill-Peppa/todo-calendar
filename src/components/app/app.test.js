import renderer from 'react-test-renderer';

import App from './app';

it('should correctly render App', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
