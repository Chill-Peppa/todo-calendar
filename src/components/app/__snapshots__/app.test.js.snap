// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render App 1`] = `
<div
  className="app"
>
  <section
    className="calendar"
  >
    <div
      className="calendar__header"
    >
      <h1
        className="calendar__title"
      >
        Июнь 2024
      </h1>
      <div
        className="calendar__button-container"
      >
        <button
          className="calendar__button"
          onClick={[Function]}
        >
          <img
            alt="left arrow"
            className="calendar__prev"
            src="left-arrow.svg"
          />
        </button>
        <button
          className="calendar__button"
          onClick={[Function]}
        >
          <img
            alt="right arrow"
            className="calendar__next"
            src="right-arrow.svg"
          />
        </button>
      </div>
    </div>
    <div
      className="calendar__main"
    >
      <ul
        className="calendar__weekdays"
      >
        <li
          className="calendar__weekday"
        >
          Пн
        </li>
        <li
          className="calendar__weekday"
        >
          Вт
        </li>
        <li
          className="calendar__weekday"
        >
          Ср
        </li>
        <li
          className="calendar__weekday"
        >
          Чт
        </li>
        <li
          className="calendar__weekday"
        >
          Пт
        </li>
        <li
          className="calendar__weekday"
        >
          Сб
        </li>
        <li
          className="calendar__weekday"
        >
          Вс
        </li>
      </ul>
      <ul
        className="calendar__days"
      />
    </div>
    <p
      className="calendar__week-deals"
    >
      Желаете посмотреть планы на неделю?
       
      <span
        className="calendar__see-more"
        onClick={[Function]}
      >
        Посмореть
      </span>
    </p>
  </section>
</div>
`;
